.travis.yml

```diff
@@ -144,12 +142,16 @@ jobs:
       php: "nightly"
       env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
 
-    - name: PHP and JavaScript unit tests (7.4, WordPress trunk, with code coverage)
-      if: branch = develop AND type = push
+    - name: PHP and JavaScript unit tests (7.4, WordPress latest, with code coverage)
       php: "7.4"
       env: WP_VERSION=latest DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1 RUN_PHPUNIT_COVERAGE=1
       before_script:
-        - echo "Running unit tests with code coverage..."
+        - phpenv config-rm xdebug.ini || echo "xdebug.ini does not exist."
+        - pecl install pcov
+        - echo "pcov.directory=." >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+        # PHPUnit has to be installed from source to allow pcov/cobbler to patch it.
+        - composer global require --dev --ignore-platform-reqs phpunit/phpunit:^7 pcov/clobber
+        - $(composer config home)/vendor/bin/pcov $(composer config home) clobber
```

Comment:
I would extract this into a script in the `bin` folder to not overload the Travis config file.